Sources/Core/Internal/Screen-macOS.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Screen+macOS.swift
+//  ImagineEngine-iOS
+//
+//  Created by Guilherme Rambo on 14/10/17.
+//  Copyright © 2017 ImagineEngine. All rights reserved.
+//
+
+#if os(OSX)
```

Comment:
You shouldn't need any conditional compilation here since this file will only be included in the macOS target.